dls-controls/malcolmjs
# Feature/link info #175

## Description

Added link info the MalcolmJS, this also has the ability to delete a link.
Added a range of performance optimisations as well.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- run the site, bring up a layout and click a link - see the info, try changing the source port as well as deleting the link with the button

## Agile board tracking

connect to #175 
connect to #235 
connect to #232 
connect to #252 

closes #175 
closes #235 
closes #232
